tests/reader/test_ncmapss.py

```diff
@@ -47,6 +47,22 @@ def test_prepare_data(should_run, mocker):
         mock_save_scaler.assert_not_called()
 
 
+
+@pytest.mark.needs_data
+@pytest.mark.parametrize("scaling_range", [(-1.0, 1.0), (0.0, 2.0)])
+def test_scaling_range(scaling_range):
+    reader = NCmapssReader(fd=1, scaling_range=scaling_range)
+    reader.prepare_data()
+    features, _ = reader.load_split("dev")
+
+    reader = NCmapssReader(fd=1, scaling_range=(0, 1))
+    reader.prepare_data()
+    features_default, _ = reader.load_split("dev")
+
+   assert not np.array_equal(features[0][:, :, 1], features_default[0][:, :, 1])
```

Comment:
I think testing min and max of the features would be a more suitable test. After scaling `features` should be inside the scaling range. Additionally, you should test all elements of `features`, not only the first one. It is okay to have an `assert` inside a `for` loop.